sdk.yaml

```diff
@@ -164,6 +164,10 @@ Realtime:
           .synopsis: |
             Enter or leave presence for a given `clientId`,
             not necessarily the id for this client.
+        Extras:
```

Comment:
According to the spec you can also `presence.leave(data, extras)` but we don't have a feature node for `presence.leave`. Might be worth either:
1. Adding "Presence: Leave", or
2. Having a single "Extras" node which encompasses enter, update, and leave